src/test/java/uk/gov/hmcts/reform/idam/service/DeleteUserServiceTest.java

```diff
@@ -36,11 +40,12 @@ class DeleteUserServiceTest {
     @Test
     void shouldMakeDeleteRequest() {
         Response response = mock(Response.class);
+        when(idamTokenGenerator.getIdamAuthorizationHeader()).thenReturn("Authorization: Bearer token");
```

Comment:
The response is slightly misleading, "Bearer 123455.." is more appropriate

Reply:
To be fair, it doesn't really matter here, it can be empty string as long as it is a string. Updated.